elements/rh-avatar/rh-avatar.ts

```diff
@@ -158,16 +158,20 @@ export class RhAvatar extends BaseAvatar {
   render() {
     return html`
       <canvas part="canvas"></canvas>
-      <img src="${ifDefined(this.src)}" alt="" part="img"/>
+      <img src="${ifDefined(this.src)}" alt="" part="img">
     `;
   }
 
   firstUpdated() {
-    this._initCanvas();
+    this.#initCanvas();
     this.dispatchEvent(deprecatedCustomEvent('rh-avatar:connected'));
   }
 
-  @bound private _initCanvas() {
+  dispatchEvent(arg0: any) {
```

Comment:
this looks like it was generated by typescript at some point, and can be removed.